app/shared/src/ReadMore.js

```diff
@@ -0,0 +1,73 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import OriginalReadMore from 'react-native-read-more-text'; // eslint-disable-line
```

Comment:
Please use this format:

```js
// eslint-disable-line no-restricted-imports
```

It will catch also other rules and it's more clear what rule is disabled here.